### PR TITLE
Dutch translations

### PR DESCRIPTION
**Pull request by kevit-development**
Thanks for contributing to Postleaf! 💙💚

### Contributor License Agreement

We want Postleaf to remain free and open for everyone to use forever, so please don’t submit any code or content that is licensed or copyrighted without first getting written consent.

Before we accept your pull request, we ask that you agree to the following terms to ensure there are no conflicts with Postleaf’s license now or in the future.

1. I certify that the code I am contributing to this project is completely original or free of any known license or copyright.

2. I agree to allow the developer of Postleaf, A Beautiful Site, LLC, license or relicense my work at their discretion. By submitting this pull request, I agree to forfeit all copyrights, intellectual property rights, and royalties pertaining to my contribution.

3. By entering my name in the space provided below, I agree to all of the terms mentioned above.

Signed: KevIT

Date: 2016-08-15

### CODE REVIEW

File: app/source/languages/nl-nl.php [hunk 1]
```diff
@@ -0,0 +1,275 @@
+?php
```
Comment (claviska) [inline app/source/languages/nl-nl.php]: Missing <

**Comment by claviska** [comment]
Hi, thanks for contributing 👍 

Looks like the first line is missing the opening `<`. Also, was this made against the latest English translation? It looks like there are about 20 terms missing.

**Comment by kevit-development** [comment]
@claviska Hey, sorry for the `<`, This was taken from the most recent "de-de.php" lanuage file, could you provide me the lines that are missing?

**Comment by claviska** [comment]
No worries. I just pushed some changes to master that remove some unused terms. If you compare to `en-us.php`you'll be up to the latest version.

**Comment by kevit-development** [comment]
@claviska I've updated the translations to the newest version of en-us.php.

**Comment by claviska** [comment]
Awesome. Looks like the last commit removed `nl-nl.php` though. Can you push the finished version so I can merge it? :)

**Pull request closed without merging**